pkg/util/metrics_helper.go

```diff
@@ -805,3 +805,23 @@ type CollectorVec interface {
 	prometheus.Collector
 	Delete(labels prometheus.Labels) bool
 }
+
+// RegisterCounterVec registers new CounterVec with given name,namespace and labels.
+// If metric was already registered it returns existing instance.
+func RegisterCounterVec(registerer prometheus.Registerer, namespace, name, help string, labels []string) *prometheus.CounterVec {
```

Comment:
Q: what's the problem in using existing `prometheus.MustRegister` helper here? I would let it panic in case of duplicate registration rather than handling it via `promtheus.AlreadyRegisteredError` handling. Am I missing any specific use case here?